packages/api-core/src/ReactionAPICore.js

```diff
@@ -107,6 +107,31 @@ const startOptionsSchema = new SimpleSchema({
 
 const listenForDeath = _.once(diehard.listen.bind(diehard));
 
+const getPubSub = () => {
+  const {
+    REDIS_PUB_SUB_URL,
+    REDIS_PUB_SUB_HOST,
+    REDIS_PUB_SUB_PORT,
+    REDIS_PUB_SUB_USERNAME,
+    REDIS_PUB_SUB_PASSWORD,
+    REDIS_PUB_SUB_DB
+  } = config;
```

Comment:
I would add some info-level logging before each of these returns so that it's clear in the logs which type of pubsub is being used.